code/stable/projectile/SRS/JSON/Projectile_SRS.ipynb

```diff
@@ -127,7 +127,7 @@
     "|IM|Instance Model|\n",
     "|PS|Physical System Description|\n",
     "|R|Requirement|\n",
-    "|RefBy|Referenced By|\n",
+    "|RefBy|Referenced by|\n",
     "|Refname|Reference Name|\n",
```

Comment:
Why does "Referenced by" have a lowercase "b" but "Reference Name" have an uppercase "N"?

Reply:
I _think_ that displaying in the final artifacts should have the capital letters, it's just that the 'backend' code needs to use the lowercase copies.

Reply:
Right, but this is in a generated artifact (the Jupyter notebook for Projectile), so we would expect a capital "B" in "Referenced By"

Reply:
Yep, exactly :smile:

Reply:
I agree. It should be "Referenced By" with a capital B. I will try and fix that issue right now!